dev-python/typing/typing-3.7.4.1.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_6 pypy )
+
+inherit distutils-r1
+
+DESCRIPTION="Type Hints for Python"
```

Comment:
I presume this includes backports from 3.7, correct?

Reply:
yes